pkg/kubelet/apis/config/types.go

```diff
@@ -363,6 +363,11 @@ type KubeletConfiguration struct {
 	Logging componentbaseconfig.LoggingConfiguration
 	// EnableSystemLogHandler enables /logs handler.
 	EnableSystemLogHandler bool
+	// ShutdownGracePeriod specifies the total duration that the node should delay the shutdown and total grace period for pod termination during a node shutdown.
```

Comment:
please mention the default value

Reply:
Also the feature flag it is guarded with

Reply:
Done.